build.go

```diff
@@ -212,6 +212,40 @@ func (b *Build) GetConsoleOutput() string {
 	return content
 }
 
+// GetProgressiveText gets progressive text with the param in previous response header.
+func (b *Build) GetProgressiveText(start int64) (string, string, error) {
+	startStr := strconv.FormatInt(start, 10)
+	url := b.Base + "/logText/progressiveText"
+
+	var content string
+	querymap := make(map[string]string)
+	querymap["start"] = startStr
+
+	response, err := b.Jenkins.Requester.Get(url, &content, querymap)
+	if err != nil {
+		return "", "", err
+	}
+	textSize := response.Header.Get("X-Text-Size")
+	return content, textSize, nil
+}
+
+// GetProgressiveHTML gets progressive html with the param in previous response header.
+func (b *Build) GetProgressiveHTML(start int64) (string, string, error) {
+	startStr := strconv.FormatInt(start, 10)
+	url := b.Base + "/logText/progressiveHtml"
+
+	var content string
```

Comment:
The common code between `GetProgressiveText()` and `GetProgressiveHTML()` can be put in a function.